jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/Slider.java

```diff
@@ -116,9 +116,9 @@ public void slideVerticalTo(int value) {
         double minValue = Double.parseDouble(thumbContainer().getAttribute("aria-valuemin"));
         double maxValue = Double.parseDouble(thumbContainer().getAttribute("aria-valuemax"));
         if (value < minValue || value > maxValue) 
-            }
-                return
-            };
+        }
```

Comment:
Guys, c'mon, compile before pushing
`if() {
 return;
}`

Reply:
its already completed.
all symbols and tokens on correct places in correct shape